### DISCUSSION TITLE
latency measurement

### DISCUSSION THREAD

**Discussion by oak-tree** [Rubberazer/JETGPIO#9] [Q&A] [answered]
I didnt find, but maybe its already on docs, can you share the latency of JetGPIO lib comparing gpiod?

**Comment by Rubberazer** [comment, marked as answer]
UPDATE 24-09-2023:

Eventually I ran some measurements, results here: [The need for speed](https://github.com/Rubberazer/JETGPIO#the-need-for-speed)

Nope, never ran any proper measurement, I know is fast because the way is done e.g. for reading/writing to pins it manipulates (almost) directly the CPU registers through mmap, but no microseconds to show at this point.
